### DISCUSSION TITLE
Installation

### DISCUSSION THREAD

**Discussion by sappy96** [humbertogontijo/homeassistant-roborock#292] [General]
Hi,

I am a bit confused by the install instructions for this.

Where it says "add this repo into your HACS..." 

Which repo am I supposed to add?

I added the Xiaomi MIoT as there is none called Roborock in HACS that I see.

A point in the right directions would be great.

**Comment by humbertogontijo** [comment]
You need to add https://github.com/humbertogontijo/homeassistant-roborock.git as a custom repository as this is not available as default in HACS
https://hacs.xyz/docs/faq/custom_repositories/

**Reply by sappy96** [reply]
Cheers

**Comment by mar1u50** [comment]
is this still working if i'm on 2023.5 that already has a roborock integration?
